applications/nrf5340_audio/src/bluetooth/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -0,0 +1,261 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bt_le_audio_tx.h"
+
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/zbus/zbus.h>
+#include <../subsys/bluetooth/audio/bap_iso.h>
+
+#include "nrf5340_audio_common.h"
+
+/* TODO: Remove when a get_info function is implemented in host */
+#include <../subsys/bluetooth/audio/bap_endpoint.h>
```

Comment:
There is a bt_bap_ep_get_info() here: https://github.com/zephyrproject-rtos/zephyr/blob/21e37498e330f0c2896fcb8f4dd6f3f10c478bb8/include/zephyr/bluetooth/audio/bap.h#L420. We could create a separate task for it though